### PR TITLE
Update: Add requireReturnDescription for valid-jsdoc (fixes #1833)

### LINKED ISSUE #1833: requireReturnDescription for valid-jsdoc
I was wondering if adding a `requireReturnDescription` option to the valid-jsdoc rule would be acceptable? The major reason for documentation enforcement is to specify types in the code. While I like having good descriptions in the documentation I can see that other people might find it a large burden in a legacy codebase.

I might be able to do this over the weekend if it's acceptable

<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/8544096-requirereturndescription-for-valid-jsdoc?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by briandipalma**

### CODE REVIEW

File: lib/rules/valid-jsdoc.js [hunk 1]
```diff
@@ -22,7 +22,8 @@ module.exports = function(context) {
 
         // these both default to true, so you have to explicitly make them false
         requireReturn = options.requireReturn === false ? false : true,
-        requireParamDescription = options.requireParamDescription === false ? false : true;
+        requireParamDescription = options.requireParamDescription === false ? false : true,
```
Comment (nzakas) [inline lib/rules/valid-jsdoc.js]: This is a bit hard to read. Just `options.requireParamDescription !==false` is much easier.

**Comment by briandipalma** [comment]
@nzakas Done.

**Comment by nzakas** [comment]
Thanks, can you please sign our CLA? http://eslint.org/CLA it basically gives us permission to use the code you submit

**Comment by briandipalma** [comment]
Done.

**Comment by nzakas** [comment]
Thanks, looks good.

**Pull request merged**